students/Vladimir12241/dom-practice/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="uk-UA">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<link rel="shortcut icon" href="img/favicon.png" type="image/x-icon">
+	<title>TMNT</title>
+</head>
+<body>
+	
+	<div class="wrapper">
+		<header class="header">
+                <div class="logo">
+                    <a class="favicon logo-link" href="#" >
+                        <img src="img/logo.png" alt="logo">
+                    </a>
+                    </div>
```

Comment:
Would you pls prettify your code a bit? E.g. by using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).

Also it could be automated - [info](https://www.robinwieruch.de/how-to-use-prettier-vscode/)